Misc/NEWS.d/next/Core and Builtins/2020-03-11-19-17-36.bpo-39939.NwCnAM.rst

```diff
@@ -0,0 +1 @@
+Added str.removeprefix and str.removesuffix methods and corresponding bytes, bytearray, and collections.UserString methods to remove affixes from a string if present. See PEP 616 for a full description. Patch by Dennis Sweeney.
```

Comment:
FYI Sphinx syntax can be used here. For example, you can add a link to the PEP:

```suggestion
Added str.removeprefix and str.removesuffix methods and corresponding
bytes, bytearray, and collections.UserString methods to remove affixes
from a string if present.
See :pep:`616` for a full description.
Patch by Dennis Sweeney.
```

I also like to fit into 80 columns, but it's maybe just a personal preference :-D